test/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -47,6 +47,18 @@ import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 		return expectedNum;
 	}
 
+	function _updateExpectedOption(index, value, expected) {
+		for (let i = 0; i < expected.length; i++) {
+			const cat = expected[i];
+			for (let j = 0; j < cat.options.length; j++) {
+				if (options[index].cat === cat.key && options[index].key === cat.options[j].key && cat.options[j].selected !== value) {
```

Comment:
Where is `options` defined? Am I blind ;)

Reply:
Options is defined [here](https://github.com/BrightspaceUI/facet-filter-sort/blob/cbusman/add_option_update_function/test/d2l-filter-dropdown/d2l-filter-dropdown.js#L8) and populated [here](https://github.com/BrightspaceUI/facet-filter-sort/blob/cbusman/add_option_update_function/test/d2l-filter-dropdown/d2l-filter-dropdown.js#L87).